### DISCUSSION TITLE
MDMA + SSRIs

### DISCUSSION THREAD

**Discussion by zelixir25** [TripSit/drugs#82] [Drug Interactions]
I think this is classified correctly in the current chart but it can be a bit controversial so I think it's important for us to provide a source. Sarparast, et al. (2022) is fairly recent and is a systematic review that covers all literature on the subject that was available at the time of its writing. Feduccia, et al., (2021) focuses more on efficacy of MDMA-AT but does provide the same conclusion which is that SSRIs inhibit the effects of MDMA but do not pose an increased risk of serotonin syndrome as some believe.

Feduccia, A. A., Jerome, L., Mithoefer, M. C., & Holland, J. (2021). Discontinuation of medications classified as reuptake inhibitors affects treatment response of MDMA-assisted psychotherapy. Psychopharmacology, 238(2), 581–588. https://doi.org/10.1007/s00213-020-05710-w

Sarparast, A., Thomas, K., Malcolm, B., & Stauffer, C. S. (2022). Drug-drug interactions between psychiatric medications and MDMA or psilocybin: a systematic review. Psychopharmacology, 239(6), 1945–1976. https://doi.org/10.1007/s00213-022-06083-y

**Comment by zelixir25** [comment]
another source

Malcolm, B., & Thomas, K. (2022). Serotonin toxicity of serotonergic psychedelics. Psychopharmacology, 239(6), 1881–1891. https://doi.org/10.1007/s00213-021-05876-x

**Comment by utaninja** [comment]
This is fantastic and I strongly agree. I'll push this to an issue to have this added.

**Comment by utaninja** [comment]
Pushed to database https://github.com/TripSit/drugs/pull/123
